doc/lua_api.txt

```diff
@@ -2471,14 +2471,14 @@ Elements
 
 ### `style[<name>;<prop1>;<prop2>;...]`
 
-* Set the style for the named element `name`.
+* Set the style for the named element(s) `name`.
```

Comment:
I think mentioning the syntax next to the element itself could be useful, either in a bullet point or a second example (e.g. `### style[name1,name2,...;prop1;prop2;...]`)